src/docbkx/chapter72-pentaho-integration.xml

```diff
@@ -15,6 +15,30 @@
 		</abstract>
 	</chapterinfo>
 
+	<section id="configure_dc_installation">
```

Comment:
Not a blocker, but when we give IDs it's important to qualify them so that there's less chance of ID clashes. So in this case I would prefix it with "pentaho_"